backend/Dockerfile

```diff
@@ -33,7 +33,11 @@ RUN git clone "https://github.com/google/nsjail" --recursive --branch 3.1 /nsjai
 
 FROM base AS build
 
-RUN apt-get -y update && apt-get install -y \
+RUN apt-get -y update && \
+    apt-get install -y \
+    libpq-dev \
+    libffi-dev \
+    libssl-dev \
```

Comment:
These new packages are necessary to build `psycopg2`.
We can't use the binary version since they won't run.